legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/inmemory/backend/api/InMemoryWorkflowJobApi.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.inmemory.backend.api;
+
+import org.finos.legend.sdlc.domain.model.version.VersionId;
+import org.finos.legend.sdlc.server.domain.api.workflow.WorkflowJobAccessContext;
+import org.finos.legend.sdlc.server.domain.api.workflow.WorkflowJobApi;
+import org.finos.legend.sdlc.server.project.ProjectFileAccessProvider;
+
+import javax.inject.Inject;
+
+public class InMemoryWorkflowJobApi implements WorkflowJobApi
+{
+    @Inject
+    public InMemoryWorkflowJobApi()
+    {
+    }
+
+    @Override
+    public WorkflowJobAccessContext getProjectWorkflowJobAccessContext(String projectId)
+    {
+        return null;
+    }
+
+    @Override
+    public WorkflowJobAccessContext getWorkspaceWorkflowJobAccessContext(String projectId, String workspaceId, ProjectFileAccessProvider.WorkspaceAccessType workspaceAccessType)
+    {
+        return null;
+    }
+
+    @Override
+    public WorkflowJobAccessContext getVersionWorkflowJobAccessContext(String projectId, VersionId versionId)
+    {
+        return null;
+    }
```

Comment:
throw UnsupportedOperationException for methods that have not been implemented

Reply:
throwing exception for methods that not been implemented